recipes/paleomix/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "paleomix" %}
+{% set version = "1.2.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MikkelSchubert/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 7c46e8652f1c560aca81d865aec5cf7dc0309f9b3ef52c23f694f582963e33d0
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - python
+    - adapterremoval
+    - samtools
+    - picard
+    - bowtie2
+    - bwa
+    - mapdamage2
+    - gatk
+    - coloredlogs
+    - pysam
+    - ruamel.yaml
+    - biopython
+    - setproctitle
+
+test:
+  commands:
+    - main.py --help
```

Comment:
This should really not be `main.py`. Using the proper `pip install` incantation would install a script named `paleomix` instead.

Reply:
This would be the case if the tool would have a bin folder with an executable called paleomix I think? Or perhaps pip should do the trick, but I never successfully used pip to install paleomix...

Reply:
Yes, `pip` takes care of this. The `setup.py` file defines an entry_point, which tells `pip` (well, setuptools) to create a wrapper script at install time.